lsqecc/pauli_rotations/circuit.py

```diff
@@ -198,7 +215,9 @@ def commute_pi_over_four_rotation(self, index: int) -> None:
             product_of_coefficients /= 1j
             if isinstance(self.ops[next_block], Measurement):
                 if product_of_coefficients.real < 0:
-                    self.ops[next_block].isNegative = not self.ops[next_block].isNegative
+                    self.ops[next_block].isNegative = not self.ops[
+                        next_block
+                    ].isNegative
```

Comment:
It looks like the lines are shrinking a little ... are you runin `black --line-length 100`?

This is what I think it was. This file should have already been formatted. Right @alexnguyenn?

Reply:
What is our default line length? I had not set up that property. I thought the shrinking was part of the black's formatting. I have now set the line length to 100 in my editor

Reply:
yep I mentioned it above, I think we would have to add another commit to revert the changes?

Reply:
I am making another commit now. With suggestions on the same branch. Shoulf have a pr up in the next few mins